code/modules/library/admin_only.dm

```diff
@@ -0,0 +1,354 @@
+#define BOOK_ADMIN_DELETE "deleted"
+#define BOOK_ADMIN_RESTORE "undeleted"
+#define BOOK_ADMIN_REPORT "reported"
+
+/obj/machinery/computer/libraryconsole/admin_only_do_not_map_in_you_fucker
```

Comment:
i wonder if we couldn't do something opposite of `/datum/unit_test/required_map_items` so we don't have to do this level of chicanery (or at least cancel out any potential problems entirely)

Reply:
I mean uh, I could check for it in maplint.
~~I thought this was more funny but that would prob be good~~

Reply:
oh right we have maplint for that stuff whoooops